RecoMET/METPUSubtraction/src/noPileUpMEtAuxFunctions.cc

```diff
@@ -18,20 +18,17 @@ int isVertexAssociated(const reco::PFCandidate& pfCandidate,
     vtxAssociationType = noPuUtils::kChNoAssoc;
     for ( PFCandToVertexAssMap::const_iterator pfCandToVertexAssociation = pfCandToVertexAssociations.begin();
```

Comment:
Chris, I wanted to see if my suggestion makes sense or if there is a better way.
The dR computation inside this method takes up too much time.